Fixed instructions on how to register using existing account (#7416)
fishtown-analytics/docs.getdbt.com: website/docs/docs/install-dbt-extension.md

If you already have a dbt account, you do not need to re-register to use the dbt extension. The dbt extension can authenticate with the dbt platform using a `dbt_cloud.yml` file. If this file is present in your `~/.dbt/` folder, then the registration flow will automatically attempt to use this
file during registration. If you do not have a `~/.dbt/dbt_cloud.yml` file downloaded, follow the directions below:

> <Expandable alt_header="For dbt accounts where Fusion is enabled">

Reviewer: ```suggestion
<Expandable alt_header="For dbt accounts with Fusion enabled">
```

5. Move the downloaded `dbt_cloud.yml` file to your `~/.dbt/` directory.
6. To update your registration in VS Code, open the command palette (`ctrl+shift+P` (Windows/Linux) or `cmd+shift+p` (macOS)), then select `dbt: Register dbt extension` to complete the registration.

</Expandable>

> <Expandable alt_header="For dbt accounts where Fusion is not enabled">

Reviewer: ```suggestion
<Expandable alt_header="For dbt accounts without Fusion enabled">
```
